### PR TITLE
Add support for labelling issues

### PR DESCRIPTION
**Pull request by MattCopenhaver**

### CODE REVIEW

**Comment by MattCopenhaver** [comment]
Question - Since you can use this to label PRs as well, how should that be handled?  Should we rely on the user to realize that PRs are a subset of issues, or should we include these methods in both places?

File: README.md [hunk 1]
```diff
@@ -970,6 +970,30 @@ ghissue.updateComment(3, {
 ghissue.deleteComment(3, callback);
 ```
 
+#### Add label(s) (POST /repos/pksunkara/hub/issues/37/labels)
+
+```js
+ghissue.addLabels(['label-name'], callback);
+```
+
+#### Replace existing labels (PUT /repos/pksunkara/hub/issues/37/labels)
```
Comment (pksunkara) [inline README.md]: `Replace All Labels`

File: README.md [hunk 2]
```diff
@@ -970,6 +970,30 @@ ghissue.updateComment(3, {
...
+
+```js
+ghissue.replaceLabels(['label-name'], callback);
```
Comment (pksunkara) [inline README.md]: `replaceAllLabels`

File: src/octonode/issue.coffee [hunk 3]
```diff
@@ -53,5 +53,35 @@ class Issue
...
+  # Add label(s)
+  # '/repos/pksunkara/hub/issues/37/labels' POST
+  addLabels: (labels, cb) ->
+    @client.post "/repos/#{@repo}/issues/#{@number}/labels", labels, (err, s, b, h) =>
+      return cb(err) if err
+      if s isnt 200 then cb(new Error("Issue addLabels error")) else cb null, b, h
+
+  # Replace labels
+  # '/repos/pksunkara/hub/issues/37/labels' PUT
+  replaceLabels: (labels, cb) ->
```
Comment (pksunkara) [inline src/octonode/issue.coffee]: `replaceAllLabels`

**Comment by pksunkara** [comment]
> Should we rely on the user to realize that PRs are a subset of issue

I would go with that.

**Comment by pksunkara** [comment]
Thanks for the contribution.

**Pull request merged**